apps/launch.html.md.erb

```diff
@@ -8,27 +8,25 @@ order: 10
 
 <%= partial "/docs/partials/v2_transition_banner" %>
 
-To create a brand new app on Fly.io, run 
+With Fly Launch, you can create a brand new app on Fly.io by running this command from the root directory of your project:
 
 ```cmd
 fly launch
 ``` 
 
-from the root directory of your project.
+If you're in a hurry to try your first launch, we also have a condensed [Hands-on](/docs/hands-on/) that launches a super-simple demo app. You might also be interested in our [language- and framework-specific launch guides](/docs/languages-and-frameworks/).
 
-If you're in a hurry to try your first launch, we also have a condensed [Hands-on](/docs/hands-on/). You may also be interested in our [language- and framework-specific launch guides](/docs/languages-and-frameworks/).
-
-## Ingredients of a ready-to-deploy Fly App
+## Ingredients for a successful Fly Launch
 
 Here are the components of a successful launch, ready for the first deployment:
 
-* a way to get a Docker image that Fly.io can use to boot up a VM with your app running
-* Fly.io-specific [configuration](/docs/reference/configuration/) for the app, in the form of a `fly.toml` file.
-* platform resources such as [public IP addresses](/docs/reference/services/), [Fly Volumes](/docs/reference/volumes), [app secrets](/docs/reference/secrets/), [Fly Postgres](/docs/postgres/) clusters, integrated resources like [Redis by Upstash](/docs/reference/redis/)
+* a way to get a Docker image that Fly.io can use to boot up a Machine with your app running
+* Fly.io-specific [configuration](/docs/reference/configuration/) for the app, in the form of a `fly.toml` file
+* optional platform resources such as [public IP addresses](/docs/reference/services/), [Fly Volumes](/docs/reference/volumes), [app secrets](/docs/reference/secrets/), [Fly Postgres](/docs/postgres/) clusters, integrated resources like [Redis by Upstash](/docs/reference/redis/)
 
 ## Framework launch scanners
 
-Depending on your project, `fly launch` may be able to look at your app's source code and get through that list, straight to a deployed Fly App, in one swoop. This is most likely for frameworks on which Fly.io has people specializing full time. Right now that's [Elixir/Phoenix](/docs/elixir/), [Laravel](/docs/laravel/), [Rails](/docs/rails/), and [Django](/docs/django/), but we have launch guides for various other [languages and frameworks](/docs/languages-and-frameworks/).
+Depending on your project, `fly launch` might be able to look at your app's source code and get through that ingredient list, straight to a deployed Fly App. This is most likely for frameworks on which Fly.io has people specializing full time. Right now that's [Elixir/Phoenix](/docs/elixir/), [Laravel](/docs/laravel/), [Rails](/docs/rails/), and [Django](/docs/django/), but we also have launch guides for other [languages and frameworks](/docs/languages-and-frameworks/).
```

Comment:
@catflydotio this might be getting into word nerd territory, but imo "might" doesn't imply something unlikely, but rather that "something is possible" and is used to express possibility. 

I've been used to using it in tech docs because it is less ambiguous than "may" which implies permission.

Reply:
but made this change anyway since we'll revisit